MBC/Source/findBladeTriplets.m

```diff
@@ -5,15 +5,26 @@
 
     NTriplets = 0;              % first initialize to zero
     Triplets = [];
-    for i = 1:length(rotFrame)  % loop through inputs/outputs
+    origDesc = Desc;
+    
+    % hack for ElastoDyn state names (remove unnecessary text in parenthesis)
+    for i=1:length(rotFrame)
+        ix = strfind(Desc{i},'(internal DOF index = ');
+        if ~isempty(ix)
+            ix2 = strfind(Desc{i},'))');
```

Comment:
Hi @bjonkman 
Are you purposely looking for two parentheses? Should we look for one parenthesis instead? Or ")," maybe? 
And do you want to keep the "unit" in the description? 

Here are some examples of matching lines: 

ED 1st tower fore-aft bending mode DOF (internal DOF index = DOF_TFA1), m
ED 1st flapwise bending-mode DOF of blade 3 (internal DOF index = DOF_BF(3,1)), m

I'm porting it to python and just want to make sure I have the logic right.

Reply:
@ebranlard , it's been so long since I updated this that I don't remember exactly. :) I think we are just looking to match strings that are equal except for the blade number. For the ED states, I want to get rid of the indices on the blade DOFs, which is why I think I'm looking for the 2 parentheses. I'm sure other logic would work, too.